lib/premailer/rails/css_helper.rb

```diff
@@ -36,14 +34,33 @@ def css_urls_in_doc(doc)
         end
       end
 
+      def load_css_with_cache(url)
+        self.cache[url] ||= load_css(url)
+      end
+
+      def cache_enabled?
+        !::Rails.env.development?
```

Comment:
Let's check for `defined?(::Rails)` before using it as described in https://github.com/fphilipe/premailer-rails/pull/201/files#r149874772.